docker-compose.yml

```diff
@@ -25,3 +25,11 @@ services:
       - "--config.file=/etc/prometheus/prometheus.yml"
     ports:
       - 9090:9090
+  nginx:
+      build: ./nginx
```

Comment:
i don't think we need a build field, we can define the nginx container like below

https://github.com/SCE-Development/cleezy/blob/c93fe36de8ec29ec41b5644f0f75ab557a46c800/docker-compose.yml#L24

Reply:
Done